service/cloudconfig/cloud_config.go

```diff
@@ -48,6 +54,53 @@ func (me *MasterExtension) VerbatimSections() []k8scloudconfig.VerbatimSection {
 	return nil
 }
 
+// masterCloudProviderConf returns Kubernetes cloud provider config for Azure.
+func (me *MasterExtension) masterCloudProviderConf() (k8scloudconfig.FileAsset, error) {
```

Comment:
Please add TODO we should make it `cloudProviderConf` as soon as we start creating masters and worker in the same subnet and SG.

Reply:
Hmm, do we plan to use same subnet?

Reply:
Did I misunderstand you? I thought you suggested merging those two guys:


* pawel-cluster-VirtualNetwork-MasterSubnet
* pawel-cluster-VirtualNetwork-WorkerSubnet

Reply:
i forgot why i suggested this... if only for simplicity, Than we can avoid this and leave as it is right now as two networks will allow us properly assign security rules for master.

I mean for master we need to open some extra ports so proper is to have it done only for master subnet. Sorry for confusion :)